cunumeric/random/generator.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from cunumeric.random.bitgenerator import XORWOW, BitGenerator
+
+
+class Generator:
+    def __init__(self, bit_generator):
+        self.bit_generator = bit_generator
+
+    def integers(
```

Comment:
(I moved my comment to a more appropriate line) can this replace the legacy `random.randint` implementation?

Reply:
Yes. That it the target. It seems that randint is used by other parts. I wanted to split the sources of potential issues: cunumeric.random errors vs other errors from tests using cunumeric.random. Though, since this is the "default generator" we could use any such as the one you proposed. Both are fine.